packages/friends/yml/en/isaac-penington.yml

```diff
@@ -365,6 +396,21 @@ documents:
               seconds: 2841
               chapters:
                 - 17
+    related_documents:
+      - id: 92d2a0ab-55d5-4fc1-aaf6-f003b172898d
+        description:
+          In her Lives and Persecutions of the Primitive Quakers, Mary Ann Kelty shows a
+          particular affinity for the life and writings of Isaac Penington, saying of
+          him":"  "Yes, good and faithful servant!--well-instructed scribe!--bringing
```

Comment:
The quotes around the colon -- I experimented with this locally, and I think we can actually just get rid of those here, I don't think we need them at all.  I'm not 100% sure why, but let's nix them.